docs/calicoctl/container.md

```diff
@@ -0,0 +1,180 @@
+
+# User guide for 'calicoctl container' commands
+> NOTE: The `calicoctl container` configuration commands are used specifically 
+> when running Calico with Docker default networking.  These commands should 
+> NOT be used when  running Calico with Kubernetes, Mesos, the Docker 
+> libnetwork driver, or other orchestrators. 
+
+This sections describes the `calicoctl container` commands.
+
+These commands can be used to manage Calico networking for Docker containers.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a 
+full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl container' commands
+
+Run `calicoctl container --help` to display the following help menu for the 
+calicoctl container commands.
+
+```
+
+Usage:
+  calicoctl container add <CONTAINER> <IP> [--interface=<INTERFACE>]
+  calicoctl container remove <CONTAINER>
+  calicoctl container <CONTAINER> ip (add|remove) <IP> [--interface=<INTERFACE>]
+  calicoctl container <CONTAINER> endpoint show
+  calicoctl container <CONTAINER> profile (append|remove|set) [<PROFILES>...]
+
+Description:
+  Add or remove containers to calico networking, manage their IP addresses and profiles.
+  All these commands must be run on the host that contains the container.
+
+Options:
+  --interface=<INTERFACE>  The name to give to the interface in the container
+                           [default: eth1]
+
+```
+
+## calicoctl container commands
+
+
+### calicoctl container add <CONTAINER> <IP> 
+
+This command allows you to add a container into the Calico network. 
+
+This command is required for any container created using default Docker 
+networking to use Calico.  This command creates a network endpoint with the 
```

Comment:
I'd rewrite the last sentence "This command creates a new network interface within the container, connects it to the Calico network, and assigns the given IP address."

Also, `<IP>` can now be any of the following: an IP address, a pool expressed as a CIDR prefix, "ipv4", "ipv6".

This doc should explain those options.